### ISSUE TITLE
Navigation is tab-able but not visible

### ISSUE THREAD

**Issue by nonsponsored** [nonsponsored/BeBestNDS#12]
### Site

realfood.gov

### Page URL

https://realfood.gov

### Type of Issue

Accessibility

### Description of Issue

When using keyboard, the navigation links are tab-able when you enter the page, but are not visible.

### Device Type (Optional)

None

### Operating System (Optional)

None

### Browser (Optional)

None

### Confirmation

- [x] I confirm this is a factual report of a technical, usability, or accessibility issue.
